### DISCUSSION TITLE
how to change language of wsa or android app?

### DISCUSSION THREAD

**Discussion by lokize** [microsoft/WSA#370] [General]
Hi, is there any way to change language of wsa or app installed?

**Comment by Y-PLONI** [comment]
What's the problem? Settings!
As with any phone or tablet.
Maybe I didn't understand your question?

**Comment by jaholme** [comment]
WSA mirrors the language selections of Windows. If you change your preferred language list in Windows Settings > Time & language > Language & region that will be reflected to android apps.
<img width="788" alt="image" src="https://github.com/microsoft/WSA/assets/27746781/5df260c2-a712-4386-996c-7317973c987c">

However, if the app is not localized in a preferred language it may be shown in the default language for the app.

**Reply by Y-PLONI** [reply]
Very nice! I didn't know it would change with the language change from Windows. Thank you!

**Comment by HiZhixiang** [comment]
Create a destination that is "C:\Users\Your username\AppData\Local\Microsoft\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForAndroid_ 8wekyb3d8bbwe\WsaClient.exe /launch wsa://com.android.settings" shortcut, and then you can open Settings, the next as in the past on the phone operation consistent.
It's worth mentioning that Android Tiramisu forces all installed software to support the adb command for the language adjustment option, which works well for most apps that already have good multi-language support.
The adb command looks like this:
adb shell settings put global settings_app_locale_opt_enabled false

**Reply by bashenk** [reply]
This was helpful, though it seems you had a typo in the setting name. It should be `settings_app_locale_opt_in_enabled`.
